Cargo.toml

```diff
@@ -13,6 +13,13 @@ Semantic version parsing and comparison.
 
 [dependencies]
 semver-parser = "0.6.1"
+regex = "0.1"
```

Comment:
so, one unfortunate aspect of this is that we just removed the regex dependency because it's too heavy. maybe we can not use it here? let' see from below...